apps/fulltextsearch.sh

```diff
@@ -6,11 +6,7 @@
 # shellcheck disable=2034,2059
 true
 # shellcheck source=lib.sh
-NC_UPDATE=1 && ES_INSTALL=1 . <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
-unset NC_UPDATE
-unset ES_INSTALL
-
-print_text_in_color "$ICyan" "Installing Elastic Search & Full Text Search on Nextcloud..."
+. <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
```

Comment:
Why did you remove `NC_UPDATE=1 && ES_INSTALL=1` from here? Do you know the impact of removing it?

Reply:
It is moved to a later point in the script.

Reply:
OK, for what reason?

Reply:
There is no good reason. I will revert that change.

Reply:
is reverted.